lib/apia/rack.rb

```diff
@@ -152,18 +152,38 @@ def triplet_for_exception(exception)
 
     class << self
 
+      # Return a plain text triplet for the given body.
+      #
+      # @param body [String]
+      # @param status [Integer]
+      # @param headers [Hash]
+      # @return [Array]
+      def plain_triplet(body, status: 200, headers: {})
+        response_triplet(body, content_type: 'text/plain', status: status, headers: headers)
+      end
```

Comment:
This looks good @mattbearman !

Would you be able to have a go at adding a plain text endpoint to the example api used in the tests for our [apia-openapi gem](https://github.com/apiaframework/apia-openapi).

https://github.com/apiaframework/apia-openapi/blob/main/examples/core_api/base.rb

It would be good to confirm we can successfully add this change into the openapi spec that is generated.

Reply:
Good idea 😄